pypeit/scripts/collate_1d.py

```diff
@@ -738,11 +449,12 @@ def main(args):
         if args.par_outfile is None:
             args.par_outfile = os.path.join(outdir, 'collate1d.par')
         print("Writing the parameters to {}".format(args.par_outfile))
-        par.to_config(args.par_outfile)
-
-        # Make sure archive dir, if specified, exists
-        if par['collate1d']['archive_root'] is not None:
-            os.makedirs(par['collate1d']['archive_root'], exist_ok=True)
+        # Gather up config lines for the sections relevant to collate_1d
+        config_lines = par['collate1d'].to_config(section_name='collate1d') + ['']
```

Comment:
Do you mind adding to the `.to_config()` function the parameter `include_descr=False`? I found it helps the readability not having all those descriptions. Thanks!